cuequivariance_torch/cuequivariance_torch/primitives/equivariant_tensor_product.py

```diff
@@ -70,9 +70,6 @@ def __init__(
         optimize_fallback: Optional[bool] = None,
     ):
         super().__init__()
-        cue.descriptors.fully_connected_tensor_product(
```

Comment:
why is deleting this safe here?

Reply:
This like has probably been copy-pasted here by accident. it has no effect and has nothing to do here